napari/utils/colormaps/colormaps.py

```diff
@@ -111,6 +118,39 @@ def _low_discrepancy_image(image, seed=0.5, margin=1 / 256):
     return image_out
 
 
+def color_dict_to_colormap(
+    color_dict, background_label=0, empty_color='black'
+):
+    """
+    Generate a color map based on the given color dictionary
+    Parameters
+    ----------
+    color_dict : dict of label and color string matches
```

Comment:
the indentations/ new lines on these doc strings don't match [PEP257](https://www.python.org/dev/peps/pep-0257/) and the [NumPy documentation guide](https://numpy.org/doc/stable/docs/howto_document.html).